### ISSUE TITLE
202 Error On msm install mopidy_skill

### ISSUE THREAD

**Issue by tparikka** [forslund/mopidy_skill#9]
New user of Picroft here so forgive me if I'm missing something obvious. I tried SSHing to Picroft as "pi/mycroft" and ran "msm install mopidy_skill" as instructed in the README.md, but the following error is returned:

`Searching for for 'mopidy_skill'...'mopidy_skill' skill was not found
msm: error 202`

I tried running the command with sudo with the same result. Has the registered name of this skill changed?

**Comment by tparikka** [comment]
After playing around a bit, it looks like the skill is now named "mopidy".

**Comment by forslund** [comment]
Ah great, sorry for the delay. Haven't had time to check in on msm here.

**Comment by nimpo** [comment]
I get this on the mark-1 which seems similar:

pi@mark_1:~ $ msm install mopidy
INFO - Best match (0.21): mycroft-spotify by forslund
SkillNotFound: mopidy
pi@mark_1:~ $

Also tried the mopidy_skill name to no avail.  Is this a mk1 issue, mopidy_skill issue or a PBCAK issue :-) . Thanks

**Comment by Celti** [comment]
The `mopidy-skills` repository had *every* skill removed from it at the end of March, and only ones passing certain tests re-added. Mopidy was not among those, as I just discovered myself doing some digging.

@forslund, are there any fundamental issues with this skill at this time? If it was just for Spotify, well, you've already got a perfectly good default skill for that — but I'm a Google Music user. :)

**Comment by forslund** [comment]
I do not believe there are any huge issues at this point. I just need to add some tests and then have it re-submitted
